docs/metrics/getting-started-observable-counter/README.md

```diff
@@ -33,13 +33,13 @@ output from the console, similar to shown below:
 <!-- markdownlint-disable MD013 -->
 ```text
 Service.Nameunknown_service:getting-started-observable-counter
-Export 16:35:25.669 16:35:25.670 observable-counter [tag1=value1;tag2=value2] LongSum, Meter: TestMeter/0.0.1
+Export 16:35:25.669 16:35:25.670 observable-counter [tag1=value1;tag2=value2] LongSum, Meter: MyMeter/0.0.1
```

Comment:
Some of the output is very outdated, and the console exporter needs to be fixed (seems there are merge issues that some changes were lost). Can be another PR.